e2eAIOK/DeNas/nlp/bert_trainer.py

```diff
@@ -35,6 +39,15 @@ def _pre_process(self):
             custom_ops_thop = customer_ops_map_thop()
             macs_thop, _ = profile(self.model, inputs=(inputs,), custom_ops=custom_ops_thop)
             logging.info("(THOP) MACs: %.2f" % (macs_thop/(1000**3)))
+        if self.cfg.is_tl:
+            self.teacher_model = ModelBuilderNLPDeNas(self.cfg)._init_extra_model(self.cfg.teacher_model, self.cfg.teacher_model_structure)
```

Comment:
I realized that this model can only be one download from huggingface by reading '_init_extra_model', in that case, can we add some pre-check or inform users with model-names not supported in huggingface? Or the teacher model is actually has a limited option list?

Reply:
Yes, the teacher model is currently a limited option list.